### DISCUSSION TITLE
Generated mocks in single file

### DISCUSSION THREAD

**Discussion by izharishaksa** [vektra/mockery#581] [Ideas]
Hi,

Is it possible to generate all mocks in the same package into a single file?

Thanks.

**Comment by LandonTClipp** [comment]
It's not possible to do this now but it is an "approved feature" awaiting implementation: https://github.com/vektra/mockery/issues/564

I don't want work on this to be done until the `packages` feature is fully released as I intend for new features to only be added to that config mode.
